example/example-config.yml

```diff
@@ -1,6 +1,6 @@
 ---
 # Source directory is here       ****Include all trailing '/' in directory definitions****
-SRC_DIR: "../src/"
+SRC_DIR: "/opt/scsim/"

```

Comment:
Why did we change this to an absolute reference?

Reply:
In the dockerfile, it copies all src to opt/scsim/ so I changed to that. And '../src' doesn't work.

Reply:
Also the new environment bam-readcount is for the bam-readcount command. It seems it has some conflicts with other dependencies so I created a new environment.

Reply:
Can we keep it in ../src and make it work?

Reply:
I change the copy path src to  /opt/scsim/src, if you really want to have relative path, then it becomes '../opt/scsim/src/'. How about that?

Reply:
It seems conda has some problems, i will fix it.

Reply:
Not quite. What i was envisioning is that if the user has the enviroments installed on their computer, then they don't _need_ to run the docker container. But if they do what to run it in docker they can. If we have the relative path point to ../opt..., then it will _only_ work in docker. That's why the relative path was appealing.